src/core/api/finder/image_search.py

```diff
@@ -5,17 +5,10 @@
 
 import datetime
 import logging
-import time
 
 import cv2
 import numpy as np
 
-
-try:
-    import Image
-except ImportError:
-    from PIL import Image
-
```

Comment:
This conditional import is required. Deleting it will break everything related to image search.